airbyte/_executors/docker.py

```
@@ -56,3 +64,46 @@ def uninstall(self) -> NoReturn:
+    def map_cli_args(self, args: list[str]) -> list[str]:
+        """Map local file paths to the container's volume paths."""
+        new_args = []
+        logger = get_global_file_logger()
+        for arg in args:
+            if Path(arg).exists():
+                print(f"Found file input path `{arg}` exists.")
+                # This is a file path and we need to map it to the same file within the
+                # relative path of the file within the container's volume.
+                for local_volume, container_path in self.volumes.items():
+                    if Path(arg).is_relative_to(local_volume):
+                        if logger or True:
```

Ruff (0.8.2)

Use `True` instead of `... or True`

Replace with `True`

(SIM222)